src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java

```diff
@@ -494,8 +490,10 @@ public String getQuestionResultStatisticsCsv(
         }
         return studentResults;
     }
-
-    protected Map<String, TeamEvalResult> getTeamResults(List<String> teamNames,
+    
+    // added Suppress warning to avoid false positive by PMD.
+    @SuppressWarnings("PMD.UnusedPrivateMethod")
```

Comment:
The explanatory comment can be on the same line, `@SuppressWarnings("PMD.UnusedPrivateMethod") // false positive by PMD` as per how it's done everywhere else.